clang/include/clang/3C/ConstraintVariables.h

```diff
@@ -288,17 +286,12 @@ class PointerVariableConstraint : public ConstraintVariable {
   // pointers.
   bool IsZeroWidthArray;
 
-  // Was this variable a checked pointer in the input program?
-  // This is important for two reasons: (1) externs that are checked should be
-  // kept that way during solving, (2) nothing that was originally checked
-  // should be modified during rewriting.
-  bool OriginallyChecked;
-
```

Comment:
Why did the above disappear if all we are doing is tidying formatting? Was it a dead variable? That would surprise me.